codebuild/bin/criterion_delta.sh

```diff
@@ -14,9 +14,42 @@
 
 set -eu
 source ./codebuild/bin/utils.sh
+# Disable PQ
+export S2N_NO_PQ=1
+export AWS_S3_BUCKET="s3://s2n-tls-logs/"
+# Limit the number of child processes in the test run
+export XDIST_WORKERS=4
+export RUST_BACKTRACE=1
 
-make install
-source $HOME/.cargo/env
-make -C bindings/rust
-S2N_USE_CRITERION=1 TOX_TEST_NAME="$INTEGV2_TEST".py make integrationv2
-S2N_USE_CRITERION=3 TOX_TEST_NAME="$INTEGV2_TEST".py make integrationv2
+export GIT_COMMIT=$(git log -n 1 --format="%h")
+export AWS_S3_REPORT_PATH="reports/${INTEGV2_TEST}/$(date +%Y%m%d_${GIT_COMMIT})"
+
```

Comment:
Do we want a proper timestamp? Is there any way this can run twice on the same day for the same commit?